core/services/ocr2/plugins/directrequestocr/aggregation_test.go

```diff
@@ -0,0 +1,100 @@
+package directrequestocr_test
+
+import (
+	"strconv"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/smartcontractkit/chainlink/core/services/ocr2/plugins/directrequestocr"
+	"github.com/smartcontractkit/chainlink/core/services/ocr2/plugins/directrequestocr/config"
+)
+
+func req(id int, result []byte, err []byte) *directrequestocr.ProcessedRequest {
+	return &directrequestocr.ProcessedRequest{
+		RequestID: []byte(strconv.Itoa(id)),
+		Result:    result,
+		Error:     err,
+	}
+}
+
+func reqS(id int, result string, err string) *directrequestocr.ProcessedRequest {
+	return req(id, []byte(result), []byte(err))
+}
+
+func TestAggregation_Successful(t *testing.T) {
```

Comment:
No test(s) are for `CanAggregate`, I know it is trivial, but better to set expectations with tests.
Also, I would name this as `TestAggregate` as it tests `Aggregate` function only.